synapse/replication/tcp/handler.py

```diff
@@ -268,11 +268,9 @@ async def on_POSITION(self, cmd: PositionCommand):
                     missing_updates,
                 ) = await stream.get_updates_since(current_token, cmd.token)
 
-                if updates:
+                for token, row in updates:
```

Comment:
Do we need to worry about batched updates? I.e. updates with multiple entries per token?

Reply:
I was hoping you would know the answer to that :/

Reply:
Looks like you do:  https://github.com/matrix-org/synapse/blob/master/synapse/replication/tcp/resource.py#L198-L220